data/mods/Aftershock/items/armor/modular_exos/modular_exoskeleton_plating.json

```
@@ -0,0 +1,450 @@
+[
+  {
+    "id": "exo_helmet_steel",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "steel exoskeleton helmet plate" },
```

```suggestion
    "name": { "str": "steel helmet plating (exosuit)" },
```
Use that naming format for all the other plating items, makes them easier to identify. I prefer the use of the word `exosuit` because its more unique in its uses  than `exoskeleton` (E.X, In game you can potentially mutate an insect exoskeleton, while the word exosuit can only refer to a suit of power armor)